src/lightning/pytorch/callbacks/model_checkpoint.py

```diff
@@ -578,13 +602,17 @@ def _format_checkpoint_name(
                 metrics[name] = torch.tensor(0)
         filename = filename.format(metrics)
 
-        if prefix:
+        if prefix is not None:
             filename = self.CHECKPOINT_JOIN_CHAR.join([prefix, filename])
 
         return filename
 
     def format_checkpoint_name(
-        self, metrics: dict[str, Tensor], filename: Optional[str] = None, ver: Optional[int] = None
+        self,
+        metrics: dict[str, Tensor],
+        filename: Optional[str] = None,
+        prefix: Optional[str] = None,
+        ver: Optional[int] = None,
```

Comment:
```suggestion
        ver: Optional[int] = None,
        prefix: Optional[str] = None,
```
to prevent breaking chnages